src/app/Authenticated.tsx

```diff
@@ -25,6 +25,7 @@ const Authenticated = () => {
             <Routes>
                 <Route element={<AppLayout />}>
                     <Route path="/" element={<Home />} />
+                    <Route path="/test/builds" element={<Builds />} />
```

Comment:
Why was this route reintroduced?

Reply:
So you could hit the page if needed for any work you were doing.